components/ArticlePageHeader.vue

```diff
@@ -0,0 +1,213 @@
+      <div class="article-page-header-right flex justify-content-end">
+        <v-share-tools class="hidden lg:flex mx-3">
+          <v-share-tools-item
+            action="share"
+            service="facebook"
+            :url="shareUrl"
+            :utm-parameters="{
+              medium: 'social',
+              source: 'facebook',
+              campaign: 'shared_facebook',
+            }"
+            @share="
+              $analytics.sendEvent('click_tracking', {
+                event_category: 'Click Tracking',
+                component: 'Article Page Header',
+                event_label: 'Social Share Facebook',
+              })
+            "
+          />
+
+          <v-share-tools-item
+            action="share"
+            service="twitter"
+            :url="shareUrl"
+            :share-parameters="{ text: shareTitle, via: 'gothamist' }"
+            :utm-parameters="{
+              medium: 'social',
+              source: 'twitter',
+              campaign: 'shared_twitter',
+            }"
+            @share="
+              $analytics.sendEvent('click_tracking', {
+                event_category: 'Click Tracking',
+                component: 'Article Page Header',
+                event_label: 'Social Share Twitter',
+              })
+            "
+          />
+          <v-share-tools-item
+            action="share"
+            service="reddit"
+            :url="shareUrl"
+            :share-parameters="{ title: shareTitle }"
+            :utm-parameters="{
+              medium: 'social',
+              source: 'reddit',
+              campaign: 'shared_reddit',
+            }"
+            @share="
+              $analytics.sendEvent('click_tracking', {
+                event_category: 'Click Tracking',
+                component: 'Article Page Header',
+                event_label: 'Social Share Reddit',
+              })
+            "
+          />
+          <v-share-tools-item
+            action="share"
+            service="email"
+            :url="shareUrl"
+            :share-parameters="{ body: shareTitle + ' - %URL%' }"
+            :utm-parameters="{
+              medium: 'social',
+              source: 'email',
+              campaign: 'shared_email',
+            }"
+            @share="
+              $analytics.sendEvent('click_tracking', {
+                event_category: 'Click Tracking',
+                component: 'Article Page Header',
+                event_label: 'Social Share Email',
+              })
+            "
+          />
+        </v-share-tools>
+        <v-flexible-link :to="`${donateUrlBase}&utm_campaign={utmCampaign}`" raw class="pr-2">
+            <Button class="article-page-header-donate-button p-button-rounded"> 
+                <span class="p-button-label">Donate</span>
+            </Button>
+        </v-flexible-link>
+        <Button icon="pi pi-bars" class="p-button p-component p-button-icon-only p-button-text p-button-rounded" @click="openSidebar" />
```

Comment:
add `-mr-2` to the class list

Reply:
also do this to the hamburger in the GothamistMainHeader

Reply:
also add "align-items-center" here in the GothamistMainHeader file
![image](https://user-images.githubusercontent.com/13803343/185978110-8daaf4c1-2a57-4c20-83cf-84908a021fd8.png)

Reply:
this final adjustment will make the headers identical